WordPress/Classes/Networking/ServiceRemoteREST.h

```diff
@@ -1,5 +1,12 @@
 #import <Foundation/Foundation.h>
 
+typedef enum
+{
+    ServiceRemoteRESTApiVersion_1_1 = 1001,
+    ServiceRemoteRESTApiVersion_1_2 = 1002,
+    ServiceRemoteRESTApiVersionDefault = ServiceRemoteRESTApiVersion_1_1,
+} ServiceRemoteRESTApiVersion;
```

Comment:
I was just reading through the code and I think I'd still prefer to limit the amount of options available to an enum rather than accept strings as an input.  Accepting strings eliminates the need to map the values, but other than that it makes the parameter a bit more flexible than I'd like it to be.

It's really not that big of a difference but it has the added value of being much stricter in allowing any value outside the intended ones.